lambdas/retrieve-cri-oauth-access-token/src/main/java/uk/gov/di/ipv/core/retrievecrioauthaccesstoken/RetrieveCriOauthAccessTokenHandler.java

```diff
@@ -87,6 +87,8 @@ public Map<String, Object> handleRequest(Map<String, String> input, Context cont
         String credentialIssuerId = null;
 
         String ipAddress = StepFunctionHelpers.getIpAddress(input);
+        String featureSet = StepFunctionHelpers.getFeatureSet(input);
```

Comment:
RequestHelpers.getFeatureSet attaches the featureSet to the logs. StepFunctionHelpers.getFeatureSet doesn't - it probably should?